src/app/components/application/program-tabs/program-tabs.component.css

```diff
@@ -6,33 +6,35 @@
   user-select: none;
   border: 1px solid #cccccc;
   transition: all 0.3s cubic-bezier(0.25, 0.8, 0.7, 1);
-  cursor: pointer;
   padding: 0.5em;
 }
 .checkbox-container h4 {
   font-size: 1.3rem;
 }
```

Comment:
@smithmd I'm not sure where these changes came from... I pulled from UAT and didn't change these myself, do you recognize this change?